<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ustawienia QGIS Server</string>
  </property>
  <widget class="QTabWidget" name="tab_widget">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>441</width>
     <height>291</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="crs_tab">
    <attribute name="title">
     <string>Układy współrzędnych</string>
    </attribute>
    <widget class="QWidget" name="crs_container" native="true">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>425</width>
       <height>255</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="crs_buttons_widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="add_crs_button">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Dodaj wybrany układ</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/plugins/gml_explorer/add.png</normaloff>:/plugins/gml_explorer/add.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QgsProjectionSelectionWidget" name="crs_widget">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="crs_tree">
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="extent_tab">
    <attribute name="title">
     <string>Rozgłaszany zasięg</string>
    </attribute>
    <widget class="QWidget" name="extent_container" native="true">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>425</width>
       <height>255</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QWidget" name="extent_buttons_widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="map_window_extent_button">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Zasięg okna mapy</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton:checked {border: 5px solid rgba(0, 170, 0, 200);
                                      border-radius: 5px;}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/plugins/gml_explorer/map_extent.png</normaloff>:/plugins/gml_explorer/map_extent.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>90</width>
             <height>90</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="project_layers_extent_button">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Zasięg wczytanych warstw</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton:checked {border: 5px solid rgba(0, 170, 0, 200);
                                      border-radius: 5px;}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/plugins/gml_explorer/layers_extent.png</normaloff>:/plugins/gml_explorer/layers_extent.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>85</width>
             <height>85</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="attributes_tab">
    <attribute name="title">
     <string>Atrybuty obiektów</string>
    </attribute>
    <widget class="QWidget" name="attributes_container" native="true">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>425</width>
       <height>255</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QWidget" name="attributes_buttons_widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="expose_attributes_button">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Prezentuj atrybuty obiektów w WMS</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton:checked {border: 5px solid rgba(0, 170, 0, 200);
                                      border-radius: 5px;}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/plugins/gml_explorer/expose_attributes.png</normaloff>:/plugins/gml_explorer/expose_attributes.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>85</width>
             <height>85</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="hide_attributes_button">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Ukryj atrybuty obiektów w WMS</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton:checked {border: 5px solid rgba(255, 0, 0, 180);
                                      border-radius: 5px;}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/plugins/gml_explorer/hide_attributes.png</normaloff>:/plugins/gml_explorer/hide_attributes.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>85</width>
             <height>85</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsProjectionSelectionWidget</class>
   <extends>QWidget</extends>
   <header>qgsprojectionselectionwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
